internal/oci/oci.go

```diff
@@ -445,6 +447,10 @@ func (r *Runtime) ReopenContainerLog(c *Container) error {
 	return impl.ReopenContainerLog(c)
 }
 
+func (r *Runtime) GetInfraCPUs() *cpuset.CPUSet {
+	return r.config.InfraContainersCPUs
+}
+
```

Comment:
I'm not sure this really makes sense in the runtime object. Either in the server config (`pkg/config/config.go`) or in a special package makes more sense to me

Reply:
I removed this function.

@haircommander Does it ok to have the `infra_ctr_cpus` parameter under the
```
[runtime]
...
infra_ctr_cpus=<something>
...
```
or should I move this parameter under the separate section(probably in the future we can have the same functionality for the memory pinning and more).